assembl/static2/js/app/utils/translations.js

```diff
@@ -885,7 +897,8 @@ const Translations = {
       privacyPolicy: "Privacy policy",
       matomoSettings: "To modify this cookie's setting please click here",
       userSessionHelper: "This cookie is necessary to maintain the user connected on Assembl.",
-      localeHelper: "This cookie is necessary to display the texts in the language of your browser or the one you selected in the navigation bar.",
+      localeHelper:
+        "This cookie is necessary to display the texts in the language of your browser or the one you selected in the navigation bar.",
```

Comment:
`This cookie is necessary in order to display the page in the language of the browser or the language selected in the navigation bar.`